includes/class-rest.php

```
@@ -463,21 +491,25 @@ public function database_cleanup( \WP_REST_Request $request ) {
 			);
 
 			$method  = $method_map[ $type ];
-			$deleted = Database_Cleanup::$method();
+			$deleted = Database_Cleanup::invoke_cleanup_method( $method );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'PerformanceOptimise\Inc\Database_Cleanup' in method 'database_cleanup'. (undefined)

(StaticAccess)